app/models/claim.rb

```diff
@@ -193,6 +193,9 @@ def submitted?
   end
 
   def submittable?
+    # TODO evaluate this again when all the screens are built for ECP journey
```

Comment:
Seems unnecessary, specs will highlight.

More importantly there aren't specs for this.